### PR TITLE
Update cert-exporter README

### PR DESCRIPTION
**Pull request by jenting**
- add example on how to monitor custom certificates
- add helm2 and helm3 instruction commands
- bump Chart version

xrefs https://github.com/SUSE/doc-caasp/issues/1006

Signed-off-by: JenTing Hsiao <jenting.hsiao@suse.com>

### CODE REVIEW

**Comment by jenting** [comment]
cc @innobead @cclhsu @maximenoel8 PTAL

**Review by innobead — APPROVED** [review]
LGTM :+1:

**Comment by innobead** [comment]
@cclhsu please review and get it out. Thanks!

File: stable/cert-exporter/README.md [hunk 1]
```diff
@@ -8,94 +8,169 @@ This chart deploys a DaemonSet to export the metrics of cert expiration on hosts
...
+$ helm install \
+      --name <RELEASE_NAME> \
+      suse/cert-exporter
+```
+
+### Using Helm 3
 
 ```bash
-$ helm install --name my-release suse/cert-exporter
+$ helm install <RELEASE_NAME> \
```
Comment (cclhsu) [inline stable/cert-exporter/README.md:24]:
```suggestion
$ helm3 install <RELEASE_NAME> \
```

File: stable/cert-exporter/README.md [hunk 2]
```diff
@@ -8,94 +8,169 @@ This chart deploys a DaemonSet to export the metrics of cert expiration on hosts
...
 ```bash
-$ helm delete my-release --purge
+$ helm delete <RELEASE_NAME> --purge
 ```
 
-The command removes all the Kubernetes components associated with the chart and deletes the release.
+### Using Helm 3
+
+```bash
+$ helm delete <RELEASE_NAME>
```
Comment (cclhsu) [inline stable/cert-exporter/README.md:41]:
```suggestion
$ helm3 delete <RELEASE_NAME>
```

File: stable/cert-exporter/README.md [hunk 3]
```diff
@@ -8,94 +8,169 @@ This chart deploys a DaemonSet to export the metrics of cert expiration on hosts
...
+         --set customSecret.enabled=true \
+         --set customSecret.certs[0].name=cert-manager \
+         --set customSecret.certs[0].namespace=cert-manager-test \
+         --set customSecret.certs[0].includeKeys="{*.crt,*.pem}" \
+         --set customSecret.certs[0].annotationSelector="{cert-manager.io/certificate-name}" \
+         suse/cert-exporter
+     ```
+   - Using Helm 3
+     ```bash
+     helm install <RELEASE_NAME> \
```
Comment (cclhsu) [inline stable/cert-exporter/README.md:86]:
```suggestion
     helm3 install <RELEASE_NAME> \
```

File: stable/cert-exporter/README.md [hunk 4]
```diff
@@ -8,94 +8,169 @@ This chart deploys a DaemonSet to export the metrics of cert expiration on hosts
...
+         --name <RELEASE_NAME> \
+         --set customSecret.enabled=true \
+         --set customSecret.certs[0].name=self-signed-cert \
+         --set customSecret.certs[0].includeKeys="{*.crt,*.pem}" \
+         --set customSecret.certs[0].labelSelector="{key=value}" \
+         suse/cert-exporter
+     ```
+   - Using Helm 3
+     ```bash
+     helm install <RELEASE_NAME> \
```
Comment (cclhsu) [inline stable/cert-exporter/README.md:108]:
```suggestion
     helm3 install <RELEASE_NAME> \
```

File: stable/cert-exporter/README.md [hunk 5]
```diff
@@ -8,94 +8,169 @@ This chart deploys a DaemonSet to export the metrics of cert expiration on hosts
...
+         --set customSecret.certs[0].includeKeys="{*.crt,*.pem}" \
+         --set customSecret.certs[0].annotationSelector="{cert-manager.io/certificate-name}" \
+         --set customSecret.certs[1].name=self-signed-cert \
+         --set customSecret.certs[1].includeKeys="{*.crt,*.pem}" \
+         --set customSecret.certs[1].labelSelector="{key=value}" \
+         suse/cert-exporter
+     ```
+   - Using Helm 3
+     ```bash
+     helm install <RELEASE_NAME> \
```
Comment (cclhsu) [inline stable/cert-exporter/README.md:133]:
```suggestion
     helm3 install <RELEASE_NAME> \
```

File: stable/cert-exporter/README.md [hunk 6]
```diff
@@ -8,94 +8,169 @@ This chart deploys a DaemonSet to export the metrics of cert expiration on hosts
...
+         --set customSecret.certs[0].includeKeys="{*.crt,*.pem}" \
+         --set customSecret.certs[0].annotationSelector="{cert-manager.io/certificate-name}" \
+         --set customSecret.certs[1].name=self-signed-cert \
+         --set customSecret.certs[1].includeKeys="{*.crt,*.pem}" \
+         --set customSecret.certs[1].labelSelector="{key=value}" \
+         suse/cert-exporter
+     ```
+    - Using Helm 3
+      ```bash
+      helm install <RELEASE_NAME> \
```
Comment (cclhsu) [inline stable/cert-exporter/README.md:164]:
```suggestion
      helm3 install <RELEASE_NAME> \
```

File: stable/cert-exporter/README.md (hunk 1 above)
Comment (jenting) [reply stable/cert-exporter/README.md:24]: All the doc use the command `helm` either helm2 or helm3
https://github.com/SUSE/doc-caasp/pull/896/files

**Review by maximenoel8 — APPROVED** [review]
LGTM

**Review by cclhsu — APPROVED** [review]

**Pull request merged**